tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_test.ts

```diff
@@ -401,5 +408,52 @@ describe('linked_time_fob_controller', () => {
         end: {step: 3},
       });
     });
+
+    it('changing start input modifies start step', () => {
+      let fixture = createComponent({
+        linkedTime: {start: {step: 1}, end: null},
+      });
+      fixture.detectChanges();
+
+      let mainDiv = fixture.debugElement.query(By.css('linked-time-fob div'));
+      mainDiv.triggerEventHandler('dblclick', {});
+      fixture.detectChanges();
+
+      let input = fixture.debugElement.query(By.css('linked-time-fob input'));
```

Comment:
Would be nice if you could write this query using mainDiv/startFob instead of the top-level fixture.

It would look something like:

const input = startFob.debugElement.query(By.css('input'));

Reply:
Yea I like that better. Done.